administrative-sdk/connection.js

```diff
@@ -279,36 +280,36 @@ class Connection {
    * @param {callback} [ecb] The callback that handles the error response.
    * @param {string} [auth] The authorization header value to pass along with the request.
    */
-  _ajaxDelete(url, cb, ecb, auth) {
-    var self = this;
+  _ajaxDelete(url, auth) {
     var request = new XMLHttpRequest();
```

Comment:
Part of converting to promises is to rewrite XMLHttpRequest to the new fetch API.